support-frontend/app/config/Configuration.scala

```diff
@@ -22,7 +20,9 @@ class Configuration {
 
   lazy val googleAuth = new GoogleAuth(config.getConfig("googleAuth"))
 
-  lazy val aws = new AwsConfig(config.getConfig("aws"))
+  lazy val aws = AwsConfig.fromConfig(config)
+
+  lazy val getAddressIOConfig = GetAddressIOConfig.fromConfig(config)
```

Comment:
I'm assuming this will throw an exception if the config is not present? Is it worth modelling this as an Option and failing loudly if not present?